features/logging/cluster_logging_operator.feature

```diff
@@ -0,0 +1,88 @@
+    Given the expression should be true> service_monitor('<%= cb.collection_type %>').port == "metrics"
+    And the expression should be true> service_monitor('<%= cb.collection_type %>').path == "/metrics"
+    And evaluation of `service('<%= cb.collection_type %>').ip` is stored in the :service_ip clipboard
+
+    Given I run curl command on the CLO pod to get metrics with:
+      | object     | <%= cb.collection_type %> |
+      | service_ip | <%= cb.service_ip %>      |
+      | token      | <%= cb.token %>           |
+    Then the step should succeed
+    And the expression should be true> @result[:response].include? (cb.collection_type == "fluentd" ? "fluentd_output_status_buffer_total_bytes": "rsyslog_action_processed")
+
+  # @auther qitang@redhat.com
+  # @case_id OCP-21907
+  @admin
+  @destructive
+  Scenario: Deploy elasticsearch-operator via OLM using CLI
+    Given logging operators are installed successfully
+
+  # @auther qitang@redhat.com
+  # @case_id OCP-22492
+  @admin @destructive
+  Scenario: Scale Elasticsearch nodes by nodeCount 2->3->4 in clusterlogging
+    Given I switch to cluster admin pseudo user
+    And I create clusterlogging instance with:
+      | remove_logging_pods | true                                                                                                     |
+      | crd_yaml            | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/scalebase.yaml |
+      | log_collector       | fluentd                                                                                                  |
+    Then the step should succeed
+    And the expression should be true> cluster_logging('instance').logstore_node_count == 2
+    Given 2 pods become ready with labels:
+      | cluster-name=elasticsearch,component=elasticsearch,es-node-client=true,es-node-data=true,es-node-master=true |
+    When I run the :patch client command with:
+      | resource      | clusterlogging                                          |
+      | resource_name | instance                                                |
+      | p             | {"spec":{"logStore":{"elasticsearch":{"nodeCount":3}}}} |
+      | type          | merge                                                   |
+    Then the step should succeed
+    And the expression should be true> cluster_logging('instance').logstore_node_count == 3
+    And 3 pods become ready with labels:
+      | cluster-name=elasticsearch,component=elasticsearch,es-node-client=true,es-node-data=true,es-node-master=true |
+    When I run the :patch client command with:
+      | resource      | clusterlogging                                          |
+      | resource_name | instance                                                |
+      | p             | {"spec":{"logStore":{"elasticsearch":{"nodeCount":4}}}} |
+      | type          | merge                                                   |
```

Comment:
Do we need clean-up for these patch commands? All scenarios need a separate clean-up. Or didyou add whole feature clean-up?

Reply:
Other than that lgtm. Please do not mark as resolved my comments. This is how I keep track of which changes remain to be reviewed.

Reply:
I think I already have clean-up steps since I've set `| remove_logging_pods | true                                                                                                 |` when I create the clusterlogging CR instance, if the patch command failed, then the scenario will fail, and the clusterlogging instance will be deleted.  Please look at https://github.com/openshift/verification-tests/blob/master/features/step_definitions/logging.rb#L251-L280 and https://github.com/openshift/verification-tests/pull/372/files/6f40b7f3b9ca9cdd9a3ef333deb231531c784c6f#diff-62bc6201426dc6e375578edc55458329R59-R62

Reply:
Sorry about marking your comments as resolved. I'll keep this in mind and won't change your comments' status later.